### PR TITLE
code formatting (for "global" and "telemetry" modules)

### PR DESCRIPTION
**Pull request by Howard-C**

### CODE REVIEW

File: telemetry/widgets/telemetrypermissiondialog.h [hunk 1]
```diff
@@ -24,10 +24,14 @@
 #include <QQmlEngine>
 #include <QWidget>
 
+//---------------------------------------------------------
+//   TelemetryPermissionDialog
+//---------------------------------------------------------
+
 class TelemetryPermissionDialog : public QQuickView
       {
```
Comment (anatoly-os) [inline telemetry/widgets/telemetrypermissiondialog.h]: The bracket needs to be moved to the previous line.

File: telemetry/telemetrysetup.h [hunk 2]
```diff
@@ -22,6 +22,10 @@
 
 #include "interfaces/abstractmodulesetup.h"
 
+//---------------------------------------------------------
+//   TelemetrySetup
+//---------------------------------------------------------
+
 class TelemetrySetup : public AbstractModuleSetup
       {
```
Comment (anatoly-os) [inline telemetry/telemetrysetup.h]: The bracket needs to be moved to the previous line.

File: telemetry/models/telemetrypermissionmodel.h [hunk 3]
```diff
@@ -24,18 +24,22 @@
 #include <QSettings>
 #include <QString>
 
+//---------------------------------------------------------
+//   TelemetryPermissionModel
+//---------------------------------------------------------
+
 class TelemetryPermissionModel : public QObject
       {
```
Comment (anatoly-os) [inline telemetry/models/telemetrypermissionmodel.h]: The bracket needs to be moved to the previous line.

File: telemetry/actioneventobserver.h [hunk 4]
```diff
@@ -27,13 +27,17 @@
 
 #include "interfaces/itelemetryservice.h"
 
+//---------------------------------------------------------
+//   ActionEventObserver
+//---------------------------------------------------------
+
 class ActionEventObserver : public QObject, public ServiceInjector<ITelemetryService>
       {
```
Comment (anatoly-os) [inline telemetry/actioneventobserver.h]: The bracket needs to be moved to the previous line.

**Comment by anatoly-os** [comment]
@Howard-C thank you for the efforts to make the code formatting consistent!

**Comment by Howard-C** [comment]
> @Howard-C thank you for the efforts to make the code formatting consistent!

No problem. I will do the suggested changes as soon as possible.

**Comment by vpereverzev** [comment]
@Howard-C thx, my friend! 
But, actually, i wondered about this comments before class declarations,
because i can't see any benefit for placing obvious comments like this :)
`//---------------------------------------------------------
//   ActionEventObserver
//---------------------------------------------------------

class ActionEventObserver`

I suppose that it's 'historically' legacy things, but tell me honestly - do you agree with this?

**Comment by Howard-C** [comment]
> ```
> //---------------------------------------------------------
> // ActionEventObserver
> //---------------------------------------------------------
> 
> class ActionEventObserver
> ```
> 
> I suppose that it's 'historically' legacy things, but tell me honestly - do you agree and what do you think about this? :)

These comments help me a lot when I'm reading code. When all classes and functions are squeezed together I find it difficult to navigate through and find the class or function I want. These comments separate and clear things up, and most importantly, the comments have a very different colour than other code in platforms and text editors, for example green (in Visual Studio) or grey (on GitHub, as you can see). So whenever I want to find something, the comments are very easy to notice and I can just read them instead of carefully searching line by line.

**Comment by vpereverzev** [comment]
@Howard-C Thx for sharing your experience!
Also it's interesting to know, what IDE do you use? Visual Studio?

**Comment by Howard-C** [comment]
Yes.

**Comment by Howard-C** [comment]
@anatoly-os All set!

**Pull request merged**